server/webapp/WEB-INF/rails.new/app/presenters/api_v2/config/environments_config_representer.rb

```diff
@@ -0,0 +1,36 @@
+##########################################################################
+# Copyright 2017 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV2
+  module Config
+    class EnvironmentsConfigRepresenter < ApiV2::BaseRepresenter
+
+      link :self do |opts|
+        opts[:url_builder].apiv2_admin_environments_url
+      end
+
+      link :doc do
+        'https://api.gocd.io/#environment-config'
+      end
+
+      collection :environments, embedded: true, exec_context: :decorator, decorator: EnvironmentConfigRepresenter
```

Comment:
I'd recommend against using the full namespace. All classes are resolved relative to the current space.

Additionally, upgrading an API is easier because it'd involve renaming just the top level namespace and everything underneath would work fine.